tools/gulp/tasks/unit-test.ts

```diff
@@ -51,7 +51,10 @@ gulp.task(':test:deps:inline', sequenceTask(':test:deps', ':inline-resources'));
 gulp.task('test', [':test:watch'], (done: () => void) => {
   new karma.Server({
     configFile: path.join(PROJECT_ROOT, 'test/karma.conf.js')
-  }, done).start();
+  }, (exitCode: number) => {
```

Comment:
Okay you can remove the handler. Seems like there's no way to terminate a watch task... which sucks.